ansible/roles/satellite_master/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+master_jar: satellite-master-0.2.0-SNAPSHOT-standalone.jar
+master_config_file: satellite-master-config.clj
```

Comment:
When this is a template input, it's name should be configurable (as it is), but the destination should have a fixed name (so that we can triage users' deployments).